plugins/kubernetes/src/components/DeploymentsAccordions/DeploymentsAccordions.tsx

```diff
@@ -165,13 +171,16 @@ const DeploymentAccordion = ({
         <PodsTable
           pods={ownedPods}
           extraColumns={[READY_COLUMNS, RESOURCE_COLUMNS]}
+          customColumns={customColumns}
```

Comment:
At this point, I would have expected to just pass in `columns={podTableColumns}`, no "custom" or "extra" words in there. The table would then see that this was undefined and use the defaults, or if it weren't default, use it entirely (instead of trying to merge anything)